asabiris/output/slack/service.py

```
@@ -185,13 +185,16 @@ async def send_files(self, body: str, atts_gen,):
 		)
 
 
-	def get_channel_id(self, client, channel_name, types="public_channel"):
+	def get_channel_id(self, client, channel_name, types=["public_channel", "private_channel"]):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)